composer.json

```diff
@@ -2,7 +2,7 @@
     "name": "woocommerce/woocommerce-gateway-stripe",
     "description": "Take credit card payments on your store using Stripe.",
     "homepage": "https://woocommerce.com/products/woocommerce-gateway-stripe/",
-		"license": "GPL-2.0",
+		"license": "GPL-3.0-or-later",
```

Comment:
I think this should be changed to `"GPL-2.0-or-later"`. Otherwise I believe this would count as a license change, and we can't do that without consent from all contributors 🙂